packages/xdl/package.json

```diff
@@ -12,11 +12,11 @@
   ],
   "scripts": {
     "start": "yarn run prepare && yarn run watch",
-    "build": "tsc --emitDeclarationOnly && gulp build",
+    "build": "tsc",
```

Comment:
We use [babel + lazy loading in expo-cli](https://github.com/expo/expo-cli/blob/7c40ae0a6ad0c59c707c9c167395c621c1adab32/packages/expo-cli/package.json#L13), have you validated that the approach hasn't increased execution time?

Reply:
I did profile the time it takes to import each module imported in `internal.ts`. The only module stands out is `Android.ts`, but the reason is that it's the first module that imports `@expo/config`.
<img width="658" alt="Screen Shot 2021-03-26 at 15 53 29" src="https://user-images.githubusercontent.com/497214/113269913-9a5cc500-92e1-11eb-97fd-cb26b79ddfd8.png">

`@expo/config` is really slow to import, but since practically any command requires it, lazy loading would only move the slowdown to a different point in the command lifecycle.
<img width="906" alt="Screen Shot 2021-04-01 at 12 01 15" src="https://user-images.githubusercontent.com/497214/113270515-3a1a5300-92e2-11eb-9c8c-801f62a80e91.png">

Reply:
Almost all of the `@expo/config` slowdown seems to come from loading `@expo/config-plugins`, which is really heavy:
<img width="1346" alt="Screen Shot 2021-04-01 at 17 56 26" src="https://user-images.githubusercontent.com/497214/113313227-b8dbb400-9313-11eb-9474-1c0f2b2515d2.png">